File: calculadora em python/calculadora.py
# Desenvolvido por Gabriel Belato

def calcular():
    print("\n*******************************")
    print("*      Menu Calculadora       *")
    print("*******************************")
    print("* 1 - Adição                 *")
    print("* 2 - Subtração              *")
    print("* 3 - Multiplicação          *")
    print("* 4 - Divisão                *")
    print("* 5 - Sair                   *")
    print("*******************************")

    try:
        opc = int(input("Digite a opção desejada: "))

        if opc < 1 or opc > 5:
            print("\nOpção Inválida!")
            return

        if opc == 5:
            print("\nSaindo do sistema.")
            return

        n1 = float(input("\nDigite o primeiro número: "))
        n2 = float(input("Digite o segundo número: "))

        if opc == 1:
            print(f"\n{n1:.2f} + {n2:.2f} = {n1 + n2:.2f}")
        elif opc == 2:
            print(f"\n{n1:.2f} - {n2:.2f} = {n1 - n2:.2f}")
        elif opc == 3:
            print(f"\n{n1:.2f} x {n2:.2f} = {n1 * n2:.2f}")
        elif opc == 4:
            if n2 == 0:
                print("\nErro: divisão por zero!")
            else:
                print(f"\n{n1:.2f} / {n2:.2f} = {n1 / n2:.2f}")

    except ValueError:
        print("\nErro: digite apenas números válidos.")

if __name__ == "__main__":
    calcular()

File: calculadora em c/calculadora.c
// Desenvolvido por Gabriel Belato

#include <stdio.h>

int main() {
    float n1, n2;
    int opc;

    printf("\n*******************************");
    printf("\n*      Menu Calculadora       *");
    printf("\n*******************************");
    printf("\n* 1 - Adição                 *");
    printf("\n* 2 - Subtração              *");
    printf("\n* 3 - Multiplicação          *");
    printf("\n* 4 - Divisão                *");
    printf("\n* 5 - Sair                   *");
    printf("\n*******************************");

    printf("\nDigite a opção desejada: ");
    scanf("%d", &opc);

    if (opc < 1 || opc > 5) {
        printf("\nOpção Inválida!\n");
    } 
    else if (opc == 5) {
        printf("\nSaindo do sistema.\n");
        return 0;
    } 
    else {
        printf("\nDigite o primeiro número: ");
        scanf("%f", &n1);
        printf("Digite o segundo número: ");
        scanf("%f", &n2);

        switch (opc) {
            case 1:
                printf("\n%.2f + %.2f = %.2f\n", n1, n2, n1 + n2);
                break;
            case 2:
                printf("\n%.2f - %.2f = %.2f\n", n1, n2, n1 - n2);
                break;
            case 3:
                printf("\n%.2f x %.2f = %.2f\n", n1, n2, n1 * n2);
                break;
            case 4:
                if (n2 == 0)
                    printf("\nErro: divisão por zero!\n");
                else
                    printf("\n%.2f / %.2f = %.2f\n", n1, n2, n1 / n2);
                break;
        }
    }

    return 0;
}